### PR TITLE
AWS Kinesis/CloudWatch docs

### PR DESCRIPTION
**Pull request by danotorrey**
Docs for AWS Kinesis/CloudWatch.

The content is very rough currently, and needs improvement. Just getting some ideas down.

### CODE REVIEW

**Comment by danotorrey** [comment]
@ceruleancee Thanks for the work on these AWS docs. Looks excellent!

I just realized that I created the branch from this PR from the wrong base branch (`3.0` instead of `3.1`). We can fix this next week on Tuesday.

File: pages/integrations/inputs/kinesis_cloudwatch_input.rst [hunk 1]
```diff
@@ -0,0 +1,112 @@
+.. _kinesis_cloudwatch_input:
+
+************************
+Kinesis/CloudWatch Input
```
Comment (danotorrey) [inline pages/integrations/inputs/kinesis_cloudwatch_input.rst]: @ceruleancee What do you think about including AWS before the input name?
```suggestion
AWS Kinesis/CloudWatch Input
```

**Review by danotorrey — COMMENTED** [review]
@ceruleancee These docs look great to me. :)

I noticed one interesting issue when sizing the page down (GIF attached - screen shots and sections aligning unevenly). I am not sure what this is from. Perhaps from the multi-column layout? @jalogisch Do you have any input?

Sep-03-2019 17-03-27

**Comment by danotorrey** [comment]
@jalogisch We are trying to deliver a link to the AWS documentation to Marketing tomorrow, so we're hoping to get this merged tomorrow. We are wondering if you have time to look at this tomorrow and provide any feedback that you have.

**Comment by jalogisch** [comment]
Yes @danotorrey you are right. AFAIK that is the reason we mostly do not use that. 

But that is nothing we should be worry about. Personal I would switch the order of manually and automatic setup. Just because that this is most people would like to have that.

**Review by jalogisch — APPROVED** [review]
👍

**Comment by LeeU1911** [comment]
@danotorrey @jalogisch I find this page quite confusing because after installing `graylog-integrations-plugins` I don't see `AWS Kinesis/CloudWatch Input` available. Is there any catch here?

**Pull request merged**